### ISSUE TITLE
feature: Create Top Players Page (Dashboard)

### ISSUE THREAD

**Issue by zyzmoz** [zyzmoz/global-player#77]
# Overview

Create the page using mock data following [designs](https://www.figma.com/file/ziqdszqhMbmj61LDNFw2E6/Supernova%3A-Global-Player?node-id=1564%3A16261)

# Acceptance Criteria

The page should be similar to the designs

# Out of Scope

Backend integration

**DEV NOTE:** Blocked by #28 #29 #105
